src/scripts/components/Play.js

```diff
@@ -106,6 +106,13 @@ const VideoWrapper = styled.div`
   }
 `
 
+const VideoWrapperContent = styled.div`
+  display: flex;
+  flex: 1 1 840px;
+  flex-direction: column;
+  margin-right: 20px;
+`
```

Comment:
Does this not screw up the embedded player?

Reply:
We really need to refactor `Play.js` so that it doesn't contains as many `isEmbed` conditionals